components/user_settings/display/manage_timezones.jsx

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedHTMLMessage, FormattedMessage} from 'react-intl';
+
+import {updateUser} from 'actions/user_actions.jsx';
```

Comment:
Not a big deal, but it would be a bit nicer to pass in the mattermost-redux update user action through an `index.js` container so we can eventually get rid of the wrapper actions like this one

Reply:
@jwilander does the redux library support migrating away from these wrapper actions?

Reply:
Sorry, missed your comment. Yeah if the actions are already in mattermost-redux then you can just pass it in directly and if you need to get results right away from the action, just `await` on it instead of passing callbacks like we do for the wrapper functions